### PR TITLE
Update code for getting data form API

### PR DESCRIPTION
**Pull request by jinety**

### CODE REVIEW

File: practice-three/src/js/script.js [hunk 1]
```diff
@@ -208,10 +185,40 @@ const handleCreateForm = () => {
...
+            <td>${element.salary}</td>
+            <td>${element.city}</td>
+            <td class="td-btn">
+              <button type="button" class="delete-button">Delete</button>
+            </td>
+          </tr>
+        `;
+      });
+
+      document.getElementById('userTableBody').innerHTML = tableTemplate;
```
Comment (phuocduong-agilityio) [inline practice-three/src/js/script.js]: As I remember, you have this element before, pls use it

File: practice-three/db.json [hunk 2]
```diff
@@ -35,61 +35,19 @@
...
-      "name": "Hoa Nguyen",
-      "email": "vhoa1000@gmail.com",
-      "salary": "123",
-      "city": "Da Nang",
-      "id": 13
+      "id": 7
     }
   ]
-}
+}
```
Comment (hieungo-agilityio) [inline practice-three/db.json]: Please don't change this file.

File: practice-three/src/js/script.js [hunk 3]
```diff
@@ -208,10 +185,40 @@ const handleCreateForm = () => {
...
+ * The function takes data from the API and displays it on a table in HTML
+ */
+ const renderUserTable = () => {
+  fetch(userApi)
+    // Parses JSON response into native JavaScript objects 
+    .then((response) => response.json())
+    .then((user) => {
+      let tableTemplate = '';
+         
+      user.forEach(element => {
```
Comment (phuocduong-agilityio) [inline practice-three/src/js/script.js]: `user` mean one record of user? element is not suitable name

File: practice-three/src/js/script.js [hunk 4]
```diff
@@ -208,10 +174,40 @@ const handleCreateForm = () => {
...
 }
 
+/**
+ * The function takes data from the API and displays it on a table in HTML
+ */
+ const renderUserTable = () => {
+  fetch(userApi)
+    // Parses JSON response into native JavaScript objects 
+    .then((response) => response.json())
+    .then((userDatabase) => {
```
Comment (phuocduong-agilityio) [inline practice-three/src/js/script.js]: userDatabase => Could you use another name?

File: practice-three/src/js/script.js [hunk 5]
```diff
@@ -208,10 +174,40 @@ const handleCreateForm = () => {
...
+      let tableTemplate = '';
+         
+      userDatabase.forEach(element => {
```
Comment (phuocduong-agilityio) [inline practice-three/src/js/script.js]: element => Could you use another name

File: practice-three/src/js/script.js [hunk 6]
```diff
@@ -208,10 +174,41 @@ const handleCreateForm = () => {
...
+      users.forEach(user => {
+        tableTemplate += ` 
+          <tr class="content-row">
+            <td>${user.name}</td>
+            <td>${user.email}</td>
+            <td>${user.salary}</td>
+            <td>${user.city}</td>
+            <td class="td-btn">
+              <button type="button" class="delete-button">Delete</button>
+              <button type="button" class="update-button">Update</button>
```
Comment (phuocduong-agilityio) [inline practice-three/src/js/script.js]: Remove this line, this function haven't implemented yet. So it can not do anything if user click on this button

**Pull request merged**